joint_trajectory_controller/package.xml

```diff
@@ -4,29 +4,26 @@
   <version>4.1.0</version>
   <description>Controller for executing joint-space trajectories on a group of joints</description>
   <maintainer email="bence.magyar.robotics@gmail.com">Bence Magyar</maintainer>
-  <maintainer email="jordan.palacios@pal-robotics.com">Jordan Palacios</maintainer>
-  <maintainer email="karsten@openrobotics.org">Karsten Knese</maintainer>
+  <maintainer email="denis.stogl@stoglrobotics.de">Dr. Denis Štogl</maintainer>
+  <maintainer email="christoph.froehlich@ait.ac.at">Christoph Froehlich</maintainer>
+  <author email="jordan.palacios@pal-robotics.com">Jordan Palacios</author>
+  <author email="karsten@openrobotics.org">Karsten Knese</author>
```

Comment:
```suggestion
```


there were so many authors to the JTC I'm not sure it's fair to only highlight a few...

The majority of it came from ROS1, then ddengster made major strides on porting it. I think it's best left without authors